core/dbt/context/providers.py

```diff
@@ -1535,7 +1542,14 @@ def env_var(self, var: str, default: Optional[str] = None) -> str:
         if return_value is not None:
             # Save the env_var value in the manifest and the var name in the source_file
             if self.model:
-                self.manifest.env_vars[var] = return_value
+                # self.manifest.env_vars[var] = return_value
```

Comment:
You didn't use the same syntax here as the other places... Are you intending to change it?